frontend/src/ModelTable.jsx

```diff
@@ -60,8 +90,54 @@ function ModelTableContents() {
       selector: (row) => row.tasks,
       cell: (row, index, column, id) =>
         row.tasks.map((t) => <TaskBadge key={t} taskName={t} />),
+    },{
+      selector: (row) => {
+        const result = modelCheckResult(row.name);
+
+        if (result === "Pass") {
+          return (
+            <img
+              src="https://img.shields.io/badge/scivision_metadata-pass-green"
+              title="The metadata for this model was successfully loaded by scivision, from the location in the catalog"
+            />
+          );
+        } else if (result === "Fail") {
+          return (
+            <img
+              src="https://img.shields.io/badge/scivision_metadata-fail-red"
+              title="Scivision metadata (yaml) file for this model failed to load or was missing at the indicated location"
+            />
+          );
+        } else {
+          return (
+            <img
+              src="https://img.shields.io/badge/scivision_metadata-unknown-lightgray"
+              title="Could not access the result for this validation check"
+            />
+          );
+        }
+      },
+      name: (
+        <span
+          className="tooltip-available"
+          title={modelValidationTimeString()}
+        >
+          Validation checks
+        </span>
+      ),
+      grow: 0.5,
     },
   ];
+  const check_models_script_url =
+   "https://github.com/alan-turing-institute/scivision/releases/download/model-checks-report-latest-release/check_models.js";
+  useScript({
+    src: check_models_script_url,
+    onReady: () => setModelChecksReport(window.global_CheckReport),
```

Comment:
should this be `window.global_CheckModelReport`

Reply:
changes addressed in last commit